### PR TITLE
Added configuration options for port forwarding

### PR DESCRIPTION
**Pull request by joostvanveelen**

### CODE REVIEW

**Comment by stixx** [comment]
Tested and worked perfectly!

File: README.md [hunk 1]
```diff
@@ -93,3 +93,21 @@ volumes:
 For more information about the different mount configuration options, see the
 [Synced Folders](https://www.vagrantup.com/docs/synced-folders/basic_usage.html) chapter within
 the Vagrant documentation.
+
+### Forwarding ports
+If you want ports from your local machine (the host) forwarded to the vagrant box (the guest), you can specify this in te configuration file.
```
Comment (niels-nijens) [inline README.md]: There's a small typo here. `te` instead of `the`.

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,9 @@
 
 # Ignore local virtual machine configuration
 config.yaml
+
+# Ignore vagrant's metadata dir
+.vagrant
+
+# Ignore ide's metadata dir
+.idea
```
Comment (niels-nijens) [inline .gitignore]: Could you remove this line? It's an ignore rule for a specific editor configuration, not really related to this project.

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,9 @@
 
 # Ignore local virtual machine configuration
 config.yaml
+
+# Ignore vagrant's metadata dir
+.vagrant
```
Comment (niels-nijens) [inline .gitignore:8]: 👍

**Review by niels-nijens — CHANGES_REQUESTED** [review]
Thanks for your contribution! I'd like to see two small changes.

File: README.md (hunk 1 above)
Comment (joostvanveelen) [reply README.md]: Fixed!

File: .gitignore (hunk 2 above)
Comment (joostvanveelen) [reply .gitignore]: it's removed.

**Review by niels-nijens — APPROVED** [review]

**Pull request merged**